src/index.ts

```diff
@@ -0,0 +1,76 @@
+import * as core from "@actions/core";
+import { exec } from "@actions/exec";
```

Comment:
Cool 😎 library that makes it _much_ easier to do shell stuff: https://github.com/google/zx#readme
You don't have to use it, I'm just letting you know this cool thing exists 😊

Reply:
I never bothered researching this, but I've always wondered why `@actions/exec` exists when there are already other popular implementations for calling commands (another one is execa). Like, is there something about `@actions/exec` that optimizes it for actions? All I've found is https://github.com/actions/toolkit/issues/315